controllers/routes/routes.js

```diff
@@ -36,11 +37,13 @@ module.exports = function(app) {
     res.redirect('/timeline/' + req.user.username);        
   });
 
+  //User logout
   app.get('/logout', function(req, res) {
     req.logout();
     res.redirect('/login');
   });
 
+  //View other User's timeline
```

Comment:
Since this is the same route for viewing your own timeline as well, can we say "View a user's timeline"?

Reply:
Yes. That would be more accurate.